content/github/managing-your-work-on-github/applying-labels-to-issues-and-pull-requests.md

```diff
@@ -20,6 +20,8 @@ versions:
 {% data reusables.repositories.select-items-in-issue-or-pr-list %}
 4. In the upper-right corner, click **Label**, then start typing the name of an existing label. Click the label's name to associate it with the selected items.
```

Comment:
```suggestion
4. In the upper-right corner, click **Label**, then start typing the name of an existing label. Click the label's name to associate it with the selected items. You can also apply a label in the Labels drop-down menu within an issue or pull request.
```

This one too 💖

Reply:
Sorry, you were very clear - I marked the comments as "resolved" before editing the file, took a call, forgot what I was doing. All requested changes committed now.